charts/cryostat/templates/NOTES.txt

```diff
@@ -59,6 +63,6 @@
 {{- else if contains "LoadBalancer" .Values.core.service.type }}
   echo http://$SERVICE_IP:{{ .Values.core.service.httpPort }}
 {{- else if contains "ClusterIP" .Values.core.service.type }}
-  http://localhost:8080
+  {{ ternary "https" "http" (or .Values.authentication.openshift.enabled .Values.oauth2Proxy.service.tls.selfSigned.enabled) }}://localhost:{{ ternary "8443" "8080" (or .Values.authentication.openshift.enabled .Values.oauth2Proxy.service.tls.selfSigned.enabled) }}
```

Comment:
Since `(or .Values.authentication.openshift.enabled .Values.oauth2Proxy.service.tls.selfSigned.enabled)` is being in many places, maybe we could refactor it into a helper named template?

Reply:
I've tried this, but I just get complaints from Helm that the `include`d template has type `string` instead of `bool` :/

Reply:
> I've tried this, but I just get complaints from Helm that the included template has type string instead of bool :/

I guess we can return string `"true"`/`"false"` and do string compare? Or perhaps just return the protocol `https`/`http`? What do you think?